### ISSUE TITLE
Setup doesn't work

### ISSUE THREAD

**Issue by wujiang** [bbinet/PyGall#1]
Hi,

Is this development still active? I tried following the README instructions but got stuck in /bin/python -m pygall.scripts.setup production.ini saying that 

"pkg_resources.VersionConflict: (WebOb 1.2b3 (/home/ubuntu/venv-pygall/lib/python2.7/site-packages), Requirement.parse('WebOb<=1.1.1'))"

It seems that the setup file is not up-to-date.

**Comment by bbinet** [comment]
I'll look at it soon and report back.

**Comment by bbinet** [comment]
That is weird, I've just tried to reproduce your issue with no success.
All the following commands worked for me:
`python virtualenv.py --no-site-packages venv`
`venv/bin/pip install pygall`
`wget https://raw.github.com/bbinet/PyGall/master/production.ini`
`venv/bin/python -m pygall.scripts.setup production.ini`

You might have an old version of WebOb already installed on your system (globally) which conflicts with pyramid 1.3 requirements.
Have you correctly setup your virtualenv with `--no-site-packages` flag to make sure you're isolated from your system python?

**Comment by wujiang** [comment]
Sorry for a late reply.

Here are what I tried (BTW, I believe when using virtualenv to setup venv, --no-site-packages is the default):

```
virtualven venv-pygall
. ./venv-pygall/bin/active
pip install pygall
wget https://raw.github.com/bbinet/PyGall/master/production.ini
python -m pygall.scripts.setup production.ini
```

Then I got this error message

> "pkg_resources.VersionConflict: (WebOb 1.2b3 (/home/ubuntu/venv-pygall/lib/python2.7/site-packages), Requirement.parse('WebOb<=1.1.1')"

As it suggested, I installed  WebOb1.1.1

```
pip install WebOb==1.1.1
```

Then I got this error message:

> "pkg_resources.VersionConflict: (WebOb 1.1.1 (/home/ubuntu/venv-pygall/lib/python2.7/site-packages), Requirement.parse('WebOb>=1.2dev'))"

It seems that I'm stuck in a loop.

**Comment by bbinet** [comment]
> Here are what I tried (BTW, I believe when using virtualenv to setup venv, --no-site-packages is the default):

Yes this is the default if your virtualenv is recent enough.

> Then I got this error message
> 
> > "pkg_resources.VersionConflict: (WebOb 1.2b3 (/home/ubuntu/venv-pygall/lib/python2.7/site-packages), Requirement.parse('WebOb<=1.1.1')"
> 
> As it suggested, I installed  WebOb1.1.1
> 
>    pip install WebOb==1.1.1
> 
> Then I got this error message:
> 
> > "pkg_resources.VersionConflict: (WebOb 1.1.1 (/home/ubuntu/venv-pygall/lib/python2.7/site-packages), Requirement.parse('WebOb>=1.2dev'))"

Which version of ubuntu do you run? (I'll try to download a livecd to
reproduce your issue)
Which version of virtualenv is used?

**Comment by bbinet** [comment]
>    virtualven venv-pygall
>    . ./venv-pygall/bin/active

Do you mean:
  . ./venv-pygall/bin/activate ?

>    pip install pygall
>    wget https://raw.github.com/bbinet/PyGall/master/production.ini
>    python -m pygall.scripts.setup production.ini

**Comment by wujiang** [comment]
- ubuntu: `11.10` with kernel `3.0.0`
- virtualenv `1.7.1.2`

Yes, it's `. ./venv-pygall/bin/activate` that I ran. Thanks.

**Comment by bbinet** [comment]
Ok, got it. This is already fixed in master (see: d16c2dd9fcd67fed98dce7abb877fd5cc26fa403), but I'll try to make a new release soon.
You can already test it from the github repository:

```
$ git clone git://github.com/bbinet/PyGall.git
$ cd PyGall
# with you venv activated:
$ (venv-pygall) python setup.py install
```

Then you can try again:

```
$ (venv-pygall) python -m pygall.scripts.setup production.ini
```

**Comment by wujiang** [comment]
Thank you for the quick fix.
